test_sets/External_UCS-related_params.txt

```diff
@@ -0,0 +1,22 @@
+./test_scripts/Polices/External_UCS/001_ATF_P_TC_PreloadedPT_without_external_consent_status_groups_struct.lua Check_that_SDL_starts_successfully_in_case_external_consent_status_groups_struct_is_omitted_in_PreloadedPT
```

Comment:
@istoimenova, Due to approved renamaing correct names are:
external_consent_status_groups/disallowed_by_external_consent_entities_off/disallowed_by_external_consent_entities_on
Please find a link to "External UCS for GENIVI email" in "[GENIVI] [Policies], [Validation] New External UCS-related params in PTU, PreloadedPT, SnapshotPT" CRQ